### PR TITLE
Hammer cli katello master release

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Requires https://github.com/theforeman/foreman-infra/pull/933

**Comment by ehelms** [comment]
Rebased and updated to new structure

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/hammer-cli-katello-master-release.yaml [hunk 1]
```diff
@@ -1,15 +0,0 @@
...
-    project-type: pipeline
-    concurrent: false
-    sandbox: false
-    concurrent: false
-    triggers:
-      - github
-    dsl:
-      !include-raw:
-        - pipelines/vars/hammer-cli-katello-master-release.groovy
-        - pipelines/release/rubygemRPMNightlyRelease.groovy
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/hammer-cli-katello-master-release.yaml]: Depending on what's merged first (https://github.com/theforeman/foreman-infra/pull/936 or this) you can remove this file as well.

Comment (ehelms) [reply puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/hammer-cli-katello-master-release.yaml]: Good eye, removed!

**Comment by ekohl** [comment]
https://ci.theforeman.org/blue/organizations/jenkins/hammer-cli-katello-master-release/detail/hammer-cli-katello-master-release/317/pipeline
```
[hammer-cli-katello] Running shell script

Using /usr/local/rvm/gems/ruby-2.4.3 with gemset hammer-cli-katello-master-release-317

rake aborted!

Don't know how to build task 'pkg:generate_source'

/usr/local/rvm/gems/ruby-2.4.3@hammer-cli-katello-master-release-317/bin/ruby_executable_hooks:15:in `eval'

/usr/local/rvm/gems/ruby-2.4.3@hammer-cli-katello-master-release-317/bin/ruby_executable_hooks:15:in `<main>'

(See full trace by running task with --trace)

script returned exit code 1
```

**Pull request merged**